docs/content/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -96,7 +96,6 @@ This allows you to deploy a standalone cluster (Session or Application Mode) in
 * manually in a local Docker setup,
 * [in a Kubernetes cluster]({{< ref "docs/deployment/resource-providers/standalone/kubernetes" >}}),
 * [with Docker Compose](#flink-with-docker-compose),
-* [with Docker swarm](#flink-with-docker-swarm).
```

Comment:
You should also remove the swarm reference [on line 38](https://github.com/apache/flink/blob/master/docs/content/docs/deployment/resource-providers/standalone/docker.md?plain=1#L38) and [line 108](https://github.com/apache/flink/blob/master/docs/content/docs/deployment/resource-providers/standalone/docker.md?plain=1#L108)

Reply:
Fixed.